packages/dnb-eufemia/src/extensions/forms/constants/locales/nb-NO.ts

```diff
@@ -34,6 +34,18 @@ export default {
       text: 'Fjern',
       confirmRemoveText: 'Er du sikker på at du vil slette dette?',
     },
+    StatusSuccess: {
+      title: 'Takk skal du ha',
+      description: 'Vi har mottatt din informasjon.',
+      buttonText: 'Tilbake til forsiden',
+    },
+    StatusError: {
+      title: 'Beklager, noe gikk galt',
+      description:
+        'Prov igjen eller ta kontakt med oss om feilen vedstår.',
```

Comment:
```suggestion
        'Prøv igjen eller ta kontakt med oss om feilen vedstår.',
```

Not sure about the "ta kontakt med oss"-text. If so, then we should perhaps provide a way of contacting us? :P

Reply:
Yeah, the original message I have seen, provides a phone number. I don't want to put that in there. But I think that this is good enough. It can vary much where and how to get in contact with support. Also, what about internal apps? Or DNB Eiendom? etc.

Reply:
Agree 🤝